avm/res/event-grid/system-topic/main.bicep

```diff
@@ -53,47 +56,16 @@ var identity = !empty(managedIdentities)
     }
   : null
 
-var builtInRoleNames = {
```

Comment:
Why was this removed? For the general role assignments **to** the system-topic, those should still  be relevant. It's only the role assignments to external resoures that don't benefit from this list.